src/project/types/book/book-render.ts

```diff
@@ -361,7 +361,7 @@ async function mergeExecutedFiles(
                 const idStr = attr.id !== "" ? `#${attr.id} ` : "";
                 const clzStr = attr.classes.map((clz) => {
                   return `.${clz} `;
-                });
+                }).join("");
                 const keyValueStr = attr.keyvalue.map((kv) => {
```

Comment:
Great find! I think this map has the exact same problem and should be fixed the same way.

Reply:
I always miss the fact that I should get a generic fix from a specific issue I found. I'll improve! 

Thanks for the review.

Reply:
Don in [16c5496](https://github.com/quarto-dev/quarto-cli/pull/1298/commits/16c54961376e13286679441643d13d1af759058d)